CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed a double Spatial component add if an Unreal component is added and destroyed in between actor's replications.
 - Fixed Actor Dormancy issue, we now correctly update if we set an actor awake and update properties on it on the same tick.
 - Fixed IsActorGroupOwnerForClass logging an error if NetDriver was not ready.
+- Fixed PlayerControllers incorrectly being deleted when Actor is teleported out of owning Server's interest range.
```

Comment:
does this mean the bug was present in 0.14 or 0.13?

Reply:
Somewhat in 0.14. The actor would be removed - but the entity wouldn't be deleted - so the effects were less impactful. It's being deleted now because of https://github.com/spatialos/UnrealGDK/pull/3292